_posts/2018-11-14-cockpit-182.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Cockpit 182
+author: Gundersanne
+date: 2018-11-14
+tags: cockpit linux
+slug: cockpit-182
+category: release
+summary: Cockpit with libvirt connection choice in VM creation and PackageKit improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 182.
+
+### Machines: Add libvirt connection choice in VM creation dialog
+
+The VM creation dialog now offers different connections to the libvirt
+daemon. The options result in either a user session URI or a system URI.
+
+![Creation dialog connection
+choice](/images/machines-libvirt-create-connection-choice.png)
+
+_Thanks to Katerina Koukiou for this improvement!_
+
+### PackageKit page improvements
+
+Update severity is now indicated by a tooltip. The severity shown is equal to
+the severity of the update with the highest severity among the available updates
+for a package.
+
+![PackageKit severity tooltip](/images/packagekit-severity-tooltips.png)
+
+Furthermore if the system is not registered the PackageKit page will now display
+this clearly.
```

Comment:
The page is called "Software Updates", not "PackageKit".

To clarify, this empty state has always existed, but only if there were really no pending updates (e. g. from repos that don't require subscription). How about:

If the system is not registered,  the Software Updates page now always points this out. This avoids confusion if there are some available updates from third-party repositories, but the system as a whole still does not receive updates.

Reply:
Ah, then I misunderstood that. I'll change it thanks!